.environment-scripts/stellar/configuration_variables.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+  
+# NCEPlibs arguments
+NCEPLIBS_PLATFORM=cheyenne
+NCEPLIBS_COMPILER=intel
+
+# ESMF arguments and environment variables
+ESMF_OS=Linux
+ESMF_COMPILER=intel
+ESMF_SITE=default
+ESMF_CC=mpicc
+
+# FMS environment variables
+FMS_CC=mpicc
+FMS_FC=mpif90
+FMS_LDFLAGS=
+FMS_LOG_DRIVER_FLAGS=
+FMS_CPPFLAGS='-Duse_LARGEFILE -DMAXFIELDMETHODS_=500 -DGFS_PHYS'
+FMS_FCFLAGS='-FR -i4 -r8'
+FMS_BATS=bat
```

Comment:
Ah right, I know this required some changes to the `setup_base_environment.sh` script as well.  Ideally we would install `bats` within the `stellar/install_base_software.sh` script too.  We should maybe talk offline about how to best handle this (and then make sure all of those changes end up in this PR).  

The easiest might be if somehow Princeton installed it for all Stellar users (then we wouldn't need to worry about this at all).  I'm not sure how amenable they might be to it, but it could be worth inquiring (see [here](https://researchcomputing.princeton.edu/systems/stellar#software) regarding software requests).  It is at least installed for all users on Gaea.

Reply:
I will inquire about installing `bats` to the system folder.

Reply:
They agreed to install "bat" into the system folder. I received an email today in the morning saying that bacula-console-bat has been installed. However, later they realzied that they did not configure the package properly. They just told me that they are going to fix this issue over the weekend.

Reply:
That's excellent news.  I'm glad they were so amenable!

Reply:
Wait, sorry I did not read your message fully; did they install the incorrect tool?  I think we want [this tool](https://github.com/bats-core/bats-core) to be installed instead, correct?

Reply:
I am going to send them another email and make sure they installed the right tool.

Reply:
Thanks!

Reply:
They did not agree to install this particular `bats` tool, so we are proceeding with installing it within `install_base_software.sh`.